install_config/storage_examples/shared_storage.adoc

```diff
@@ -11,23 +11,27 @@
 toc::[]
 
 == Overview
-The following use case describes how a user wanting to leverage shared storage for use by two seperate containers would configure the solution. This example highlights the use of NFS, but can easily be adapted to other shared storage types, such as GlusterFS. In addition, this example will show configuration of pod security as it relates to shared storage.
-
-For an explanation of persistent volumes (PVs), persistent volume claims (PVCs), and
-using NFS as persistent storage, please see the section on
+The following use case describes how a user wanting to leverage shared storage
+for use by two separate containers would configure the solution. This example
+highlights the use of NFS, but can easily be adapted to other shared storage
+types, such as GlusterFS. In addition, this example will show configuration of
+pod security as it relates to shared storage.
+
+For an explanation of persistent volumes (PVs), persistent volume claims (PVCs),
+and using NFS as persistent storage, see the section on
 link:../persistent_storage/persistent_storage_nfs[Persistent Storage Using NFS].
-This topic shows an end-to-end example of using an existing NFS cluster as an OpenShift persistent store.
-It is assumed that a working NFS server and exports have already been setup. 
-
+This topic shows an end-to-end example of using an existing NFS cluster as an
+{product-title} persistent store. It is assumed that a working NFS server and
+exports have already been setup.
```

Comment:
@ahardin-rh I know this was already here when you got here, but the last sentence might need rewording 'cos expletive and "setup" is a typo. Maybe something like:

"This topic shows and end-to-end example of using an existing NFS cluster and {product-title} persistent store, and assumes an existing NFS server and exports exist in your {product-title} infrastructure." 

Though I'm not sure what 'exports' are...